be/app/Models/User.php

```diff
@@ -56,4 +57,16 @@ public function user_words()
     {
         return $this->hasMany(User_word::class);
     }
+
+    public function UpdateAvatar($image)
```

Comment:
- variable or function name should be snakeCase
read this in your spare time https://github.com/alexeymezenin/laravel-best-practices